test/functional/formatting_stack/kondo_classpath_cache.clj

```diff
@@ -0,0 +1,21 @@
+(ns functional.formatting-stack.kondo-classpath-cache
+  (:require
+   [clojure.test :refer [are deftest is testing]]
+   [formatting-stack.kondo-classpath-cache :as sut]
+   [formatting-stack.util :refer [rcomp]]))
+
+(deftest runner
+  (testing "All choices have identical arglists (and therefore can be swapped transparently)"
+    (let [arglists (->> sut/runner-mapping vals (map (rcomp find-var meta :arglists)))]
```

Comment:
afaik `arglists` are not guaranteed to be the actual arglisting, as it's mainly used for documentation purposes. `eduction` is one of those examples:
![image](https://user-images.githubusercontent.com/4689114/112044707-6b5c8b80-8b4a-11eb-95a7-c9770bcc82e7.png)

However; I'm not aware of a better way to do so, and given the usecase it seems fine.